mlflow/tracing/fluent.py

```diff
@@ -98,11 +102,10 @@ def start_span(
     from opentelemetry import trace as trace_api
 
     from mlflow.tracing.types.wrapper import MLflowSpanWrapper, NoOpMLflowSpanWrapper
-    from mlflow.tracing.utils import get_caller_module
 
     # TODO: refactor this logic
     try:
-        tracer = _get_tracer(get_caller_module())
```

Comment:
note: It seems the module name passed to the tracer is not really important when using a single tracer instance throughout the application.